### DISCUSSION TITLE
Get SharePoint list into Bot Framework Composer and use it

### DISCUSSION THREAD

**Discussion by kwoxer** [microsoft/BotFramework-Composer#8467] [Q&A]
I am not able to find a simple way to **integrate** my SharePoint example list into my bot:

[![SharePoint list][1]][1]

I wonder if that is possible without another database like **Cosmos DB**.

What is the best practice to use such a SharePoint list and being able to search it in the Bot Framework Composer directly?

  [1]: https://i.stack.imgur.com/RtR9q.png

**Comment by guitarguySD** [comment]
Sharepoint has a REST api to get list items or manipulate lists. Google sharepoint api to get list items
